src/opening-period/page/EditOpeningPeriodPage.test.tsx

```diff
@@ -309,6 +372,84 @@ describe(`<EditNewOpeningPeriodPage />`, () => {
     });
   });
 
+  it('should render correct time-span-group rules', async () => {
```

Comment:
Should we also test that user is able to add and delete time-span-group rules?

Reply:
Maybe it should be done in the e2e-test?

Reply:
In a way, add is already tested in the e2e-test for new date-period

Reply:
- [x] Add delete rule test

Reply:
It feels kind of a corner case, and I've usually preferred to test those in unit tests, in order to not increase the amount of e2e tests because I've felt them more expensive with potential to random errors etc.. But Im not sure, you can decide